package/freescale-imx/firmware-imx/Config.in

```diff
@@ -54,21 +54,33 @@ config BR2_PACKAGE_FIRMWARE_IMX_NEEDS_DDR_FW
 	default y if BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8MM
 	default y if BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8MN
 	default y if BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8MP
+	default y if BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX93
 
 if BR2_PACKAGE_FIRMWARE_IMX_NEEDS_DDR_FW
 
 choice
-	bool "DDR training binaries"
-	default BR2_PACKAGE_FIRMWARE_IMX_LPDDR4
+	bool "DDR firmware binaries"
+	default BR2_PACKAGE_FIRMWARE_IMX_LPDDR4_PMU_TRAIN if BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8M
+	default BR2_PACKAGE_FIRMWARE_IMX_LPDDR4_PMU_TRAIN if BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8N
```

Comment:
should this be IMX8MN? trying to make to the NEEDS DDR_FW and these cpu selections don't match - not sure I am reading this right?

wouldn't the IMX8M, IMX8MM, IMX*MN, IMX8MP, IMX93 be in this default clause?

Reply:
Thank you for spotting this.